README.md

```diff
@@ -9,6 +9,7 @@ or its latest stable version.
 [![License](https://poser.pugx.org/pugx/badge-poser/license.svg)](https://packagist.org/packages/pugx/badge-poser)
 [![Downloads](https://poser.pugx.org/pugx/badge-poser/d/total.svg)](https://packagist.org/packages/pugx/badge-poser)
 [![composer.lock available](https://poser.pugx.org/pugx/badge-poser/composerlock)](https://packagist.org/packages/pugx/badge-poser)
+[![Coverage Status](https://coveralls.io/repos/github/PUGX/badge-poser/badge.svg?branch=code_coverage_report)](https://coveralls.io/github/PUGX/badge-poser?branch=code_coverage_report)
```

Comment:
the `branch=` query parameter (copy/pasted this from coveralls directly) should maybe pointed to `master`?
WDYT?

Reply:
For me `master` is the correct branch.

Reply:
Updated to `mater`.